src/bootstrap/bin/rustdoc.rs

```diff
@@ -12,12 +12,12 @@ fn main() {
     let args = env::args_os().skip(1).collect::<Vec<_>>();
     let rustdoc = env::var_os("RUSTDOC_REAL").expect("RUSTDOC_REAL was not set");
     let libdir = env::var_os("RUSTDOC_LIBDIR").expect("RUSTDOC_LIBDIR was not set");
-    let stage = env::var("RUSTC_STAGE").expect("RUSTC_STAGE was not set");
+    let stage = env!("RUSTC_STAGE","RUSTC_STAGE was not set").to_String();
     let sysroot = env::var_os("RUSTC_SYSROOT").expect("RUSTC_SYSROOT was not set");
     let mut has_unstable = false;
 
     use std::str::FromStr;
-
+    
```

Comment:
Please do not add trailing whitespace. Investigate how to configure your editor to prevent this.